packages/components/nodes/documentloaders/Playwright/Playwright.ts

```diff
@@ -121,6 +129,14 @@ class Playwright_DocumentLoaders implements INode {
                 optional: true,
                 additionalParams: true
             },
+            {
```

Comment:
can get get rid of this, but instead use a process.env to allow user to specify if needed. For example: `process.env.PLAYWRIGHT_EXECUTABLE_FILE_PATH`